minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -246,7 +244,8 @@ public void start(TachyonConf conf) throws IOException {
     // Disable hdfs client caching to avoid file system close() affecting other clients
     System.setProperty("fs.hdfs.impl.disable.cache", "true");
 
-    startMaster(conf);
+    MasterContext.getConf().merge(conf);
```

Comment:
The method start() and start(TachyonConf conf) still exist. Is that intentional? There is also a getWorkerTachyonConf() method. Now, with master context and worker context, do we need all of those methods that return or take in TachyonConf? Thanks.